arlo-client/src/components/CreateAudit.tsx

```diff
@@ -21,8 +23,19 @@ const Wrapper = styled.div`
   }
 `
 
+interface IElection {
+  id: string
+  name: string
+  date: string
```

Comment:
This is a bit ambiguous. Is it the date of the election? The date the record was created? It's a bit hard to tell from this definition.